### DISCUSSION TITLE
Problem with React native with WalletConnect 1v

### DISCUSSION THREAD

**Discussion by daryaleushkina** [WalletConnect/walletconnect-monorepo#2231] [Developer Support]
Hello. I'm using `"@walletconnect/react-native-dapp": "^1.7.8"` and Mumbai testnet: https://rpc-mumbai.maticvigil.com
I have a problem when I try to do connector.sendTransaction(tx).
But here's what happens 9 out of 10 times. I go to MetaMask and approve the transaction, and I get some hash in response, but in fact, the transaction does not go to the blockchain and is not visible later in the MetaMask mobile application.
The structure of the transaction itself is exactly correct because we use the same methods in the web version and there are no problems with the MetaMask.

Here example transaction: `{
    "value": 0,
    "chainId": 80001,
    "from": "my_address",
    "nonce": 73,
    "gasPrice": 6000000000,
    "gas": 50000,
    "to": "my_conract_address",
    "data": "0x095ea7b3000000000000000000000000b4078fca915fa2099ee2ba049043d2084d5863560000000000000000000000000000000000000000000000000de0b6b3a7640000"
}`

**Comment by duybuikardiachain** [comment]
set nonce to null, then try again
